### ISSUE TITLE
built wheel is not complete

### ISSUE THREAD

**Issue by Steph71** [tensorflow/tensorflow#45336]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows Server 2019
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: NA
- TensorFlow installed from (source or binary): Source
- TensorFlow version: 2.3.1
- Python version: 3.7.7.1
- Installed using virtualenv? pip? conda?: NA
- Bazel version (if compiling from source): 3.1.0
- GCC/Compiler version (if compiling from source): 
- CUDA/cuDNN version: 10.1
- GPU model and memory: Gforce MX940

**Describe the problem**

I Build Tensorflow 2.3.1 from source, because my graphic card (Compute Capabiliy 5) is not anymore supported by the binary releases. The Build process as such works ok, but when building the wheel and I install it to another system I miss some components.

e.g all the models under keras.applications are just not part of the wheel.

**Provide the exact sequence of commands / steps that you executed before running into the problem**
Basically I follow the instructions from here:

https://www.tensorflow.org/install/source_windows

bazel build --config=opt --config=cuda //tensorflow/tools/pip_package:build_pip_package --define=no_tensorflow_py_deps=true

bazel-bin\tensorflow\tools\pip_package\build_pip_package ..\tensorflow_pkg

The wheel is created and it is functional ... just parts of the python sources is missing

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

Please let me know which logs you need, and I will provide.

**Comment by mihaimaruseac** [comment]
Are you missing some APIs or just some source files that exist in the repo don't get exposed in the wheel?

**Comment by Steph71** [comment]
> 
> 
> Are you missing some APIs or just some source files that exist in the repo don't get exposed in the wheel?

As indicated above I think the API is complete but I am missing python sources e.g. whatever is under:

https://github.com/tensorflow/tensorflow/tree/r2.3/tensorflow/python/keras/applications

**Comment by mihaimaruseac** [comment]
I don't think they are included in the pip package, by design.

**Comment by Steph71** [comment]
@mihaimaruseac

The  [wheel](https://files.pythonhosted.org/packages/5d/6a/9669836f813b73fe5abf5e9f118ccc9b7fb060f02789d385825b0943f9c8/tensorflow-2.3.1-cp37-cp37m-win_amd64.whl) distributed through pypi has these files ...

![grafik](https://user-images.githubusercontent.com/8713992/101604609-71968400-3a01-11eb-9954-7e7a32fc77f0.png)

**Comment by mihaimaruseac** [comment]
That's the macos wheel, the bug is about a windows wheel.

Windows wheels are smaller because of maximum size limits under windows (we cannot create a zip file larger than 2GB at one point during the build).

**Comment by Steph71** [comment]
@mihaimaruseac
> That's the macos wheel, the bug is about a windows wheel.

True, Sorry for the confusion. However also the windows wheels have the files (see updated screenshot above)

**Comment by mihaimaruseac** [comment]
What commit are you building from?

**Comment by Steph71** [comment]
Tag 2.3.1-- Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 10.12.20, 18:09 schrieb Mihai Maruseac <notifications@github.com>:

   
  What commit are you building from?

**Comment by mihaimaruseac** [comment]
This is weird. We are using the exact same command.

Can you change

```
bazel-bin\tensorflow\tools\pip_package\build_pip_package ..\tensorflow_pkg
```

to have a path argument that does not share prefix with the path to the TF sources?

Alternatively, can you also try on the 2.4 branch?

**Comment by Steph71** [comment]
actually I now had a closer look into the created wheel ... and the files are actually there ...
trying to figure out why they haven't been installed in the first place ... could'nt reproduce ...

I have to appologize for creating the noise here ... not sure what happended
